### PR TITLE
fix: hide default spin button on number field

### PR DESCRIPTION
**Pull request by chrisdholt**
# Description
Currently the browser adds spin buttons to inputs with type of number. These buttons only support minimal styling and when coupled with certain MSFT design system changes, they look awful. This change serves to fix the visual problem. The longer term solution is proposed with #2202 

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [ ] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

**Comment by bheston** [comment]
I think the spinners are desirable to have by default on this control. There have been some rendering issues in the browser, but it seems normal now and looks alright by default in Explorer. Can you post an example of what's not working?

**Comment by chrisdholt** [comment]
> I think the spinners are desirable to have by default on this control. There have been some rendering issues in the browser, but it seems normal now and looks alright by default in Explorer. Can you post an example of what's not working?

These don't theme according to our system and they are squished in density driven scenarios. @Falkicon can provide more insight as he brought the issue to my attention.

Screen Shot 2019-08-30 at 9 52 08 AM

**Comment by mdtauk** [comment]
> I think the spinners are desirable to have by default on this control. There have been some rendering issues in the browser, but it seems normal now and looks alright by default in Explorer. Can you post an example of what's not working?

I don't think the idea is to not have spinners when you add the control to the page, but to suppress the browser's buttons, and use custom ones which adapt better to density, and touch usage.

File: packages/fast-components-styles-msft/src/number-field/index.ts [hunk 1]
```diff
@@ -13,6 +13,11 @@ const styles: ComponentStyles<NumberFieldClassNameContract, DesignSystem> = {
             format("0 6px 0 {0}", horizontalSpacing()),
             format("0 {0} 0 6px", horizontalSpacing())
         ),
+        "&::-webkit-inner-spin-button": {
+            appearance: "none",
+            "-webkit-appearance": "none",
+            "-moz-appearance": "none",
```
Comment (janechu) [inline packages/fast-components-styles-msft/src/number-field/index.ts]: Firefox does not support `-webkit-inner-spin-button`.
Theoretically assigning it an appearance within this selector will not affect how the control looks on firefox.

Comment (chrisdholt) [reply packages/fast-components-styles-msft/src/number-field/index.ts]: Yeah, I'll get this updated. I saw that it was changing the appearance in FF. Still need to land this with others.

**Comment by janechu** [comment]
smoked, nice solution

**Pull request merged**